jsk_perception/sample/sample_probability_image_classifier.launch

```diff
@@ -0,0 +1,63 @@
+<launch>
+
+  <arg name="gui" default="true" />
+  <arg name="GPU" default="0" />
+
+  <node name="image_publisher"
+        pkg="jsk_perception" type="image_publisher.py">
+    <rosparam subst_value="true">
+      publish_info: false
+      file_name: $(find jsk_perception)/sample/object_detection_example_1.jpg
+    </rosparam>
+  </node>
+
+  <include file="$(find jsk_perception)/launch/fcn_object_segmentation.launch">
+    <arg name="INPUT_IMAGE" value="image_publisher/output" />
+    <arg name="GPU" value="$(arg GPU)" />
+  </include>
+
+  <node name="probability_image_classifier"
+      pkg="jsk_perception" type="probability_image_classifier.py">
+    <remap from="~input" to="fcn_object_segmentation/output/proba_image" />
```

Comment:
Ah, I misunderstood by the listed `~target_names`.
Then you can do `<remap from="~target_names" to="fcn_object_segmentation/label_names" />`.